tests/unit/chroma_api/test_query_scaling.py

```diff
@@ -248,5 +248,4 @@ def test_filesystem_targets(self):
         self.assertEqual(target_scaling.queries_per_object, QUERIES_PER_TARGET)
 
         filesystem_scaling = self._measure_scaling(self._create_filesystem_n_osts, FilesystemResource, TargetResource)
-        self.assertIsInstance(filesystem_scaling, OrderN)
-        self.assertEqual(filesystem_scaling.queries_per_object, QUERIES_PER_FILESYSTEM_TARGET)
+        self.assertIsInstance(filesystem_scaling, Order1)
```

Comment:
Is this metric related?

Reply:
yeah, apparently removing the metric queries improved the runtime.

Reply:
cool 👍